Shokofin/Sync/UserDataSyncManager.cs

```diff
@@ -233,6 +238,11 @@ public async void OnUserDataSaved(object sender, UserDataSaveEventArgs e)
                     }
                 }
             }
+            catch (HttpRequestException ex) when (ex.StatusCode is HttpStatusCode.Unauthorized) {
+                TryGetUserConfiguration(e.UserId, out var userConfig);
+                if (userConfig is not null)
+                    Logger.LogError(ex, "Invalid or expired API token used. In the plugin settings, please reset the connection to Shoko Server in the user settings section (Jellyfin User={Username})", UserManager.GetUserById(userConfig.UserId).Username);
```

Comment:
We now have two places we need to update the message if we want to change it. Instead we can just re-use the thrown message and append the Jellyfin username.

(I also 1. renamed the debug variable in the message because a. the key/name had a space in it and b. because the key/name was too long IMO, 2. made sure we expect the edge case of the JF user being deleted and the settings still renaming (by adding a null check), and 3. added the user id for easier (for me at least) debugging.)
```suggestion
                    Logger.LogError(ex, "I{Message} (Username={Username},Id={UserId})", ex.Message, UserManager.GetUserById(userConfig.UserId)?.Username, userConfig.UserId);
```

Reply:
Used a seperate message in this case to make it more clear to users that it's specific to the per jellyfin user section. Have tweaked the root PUSH/GET exception message to account for this.